tasks/custom_rubo_cops/wrong_migration_version.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../../core/lib/spree/core/version'
+
+module CustomRuboCops
```

Comment:
Can we use the Solidus namespace as module namespace for this cop? I would like the output to be solidus specific

Reply:
Done and rebased. Output now looks like:

```
Solidus/WrongMigrationVersion: Subclasses of ActiveRecord::Migration must use a migration version of <= 7.0
```

Reply:
Nice! Thanks